packages/console/src/pages/Connectors/components/ConnectorName/TrialTag.tsx

```diff
@@ -0,0 +1,17 @@
+import { useTranslation } from 'react-i18next';
+
+import { Tooltip } from '@/components/Tip';
+
+import * as styles from './TrialTag.module.scss';
+
+const TrialTag = () => {
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+
+  return (
+    <Tooltip isLineEllipsisDisabled content={<div>{t('connectors.connector_trial_tip')}</div>}>
```

Comment:
`isLineEllipsisDisabled` is verbose and unclear to understand the use case

Reply:
updated with `isOverflowTruncated`

Reply:
maybe "shouldTruncate". and actually i couldn't figure out in which scenario we would like to truncate tooltip..

Reply:
Remove the 'truncated' styles since it seems redundant.